test/byContract/IexecPoco/IexecPoco1.test.ts

```diff
@@ -0,0 +1,213 @@
+const volume = 321;
+const taskIndex = 0;
+
+describe('IexecPoco1', () => {
+    let proxyAddress: string;
+    let [iexecPoco, iexecPocoAsRequester]: IexecInterfaceNative[] = [];
+    let iexecWrapper: IexecWrapper;
+    let [appAddress, datasetAddress, workerpoolAddress]: string[] = [];
+    let [
+        iexecAdmin,
+        requester,
+        sponsor,
+        beneficiary,
+        appProvider,
+        datasetProvider,
+        scheduler,
+        anyone,
+    ]: SignerWithAddress[] = [];
+    let ordersActors: OrdersActors;
+    let ordersAssets: OrdersAssets;
+    let ordersPrices: OrdersPrices;
+
+    beforeEach('Deploy', async () => {
+        // Deploy all contracts
+        proxyAddress = await loadHardhatFixtureDeployment();
+        // Initialize test environment
+        await loadFixture(initFixture);
+    });
+
+    async function initFixture() {
+        const accounts = await getIexecAccounts();
+        ({
+            iexecAdmin,
+            requester,
+            sponsor,
+            beneficiary,
+            appProvider,
+            datasetProvider,
+            scheduler,
+            anyone,
+        } = accounts);
+        iexecWrapper = new IexecWrapper(proxyAddress, accounts);
+        ({ appAddress, datasetAddress, workerpoolAddress } = await iexecWrapper.createAssets());
+        iexecPoco = IexecInterfaceNative__factory.connect(proxyAddress, anyone);
+        iexecPocoAsRequester = iexecPoco.connect(requester);
+        ordersActors = {
+            appOwner: appProvider,
+            datasetOwner: datasetProvider,
+            workerpoolOwner: scheduler,
+            requester: requester,
+        };
+        ordersAssets = {
+            app: appAddress,
+            dataset: datasetAddress,
+            workerpool: workerpoolAddress,
+        };
+        ordersPrices = {
+            app: appPrice,
+            dataset: datasetPrice,
+            workerpool: workerpoolPrice,
+        };
+        // TODO check why this is done in 00_matchorders.js
+        // await Workerpool__factory.connect(workerpoolAddress, scheduler)
+        //     .changePolicy(35, 5)
+        //     .then((tx) => tx.wait());
+    }
+
+    // TODO
+    describe('Verify signature', () => {});
+    describe('Verify presignature', () => {});
+    describe('Verify presignature or signature', () => {});
+
+    describe('Match orders', () => {
+        it('Should match orders with all assets (Standard)', async () => {
+            const { orders } = buildOrders({
+                assets: ordersAssets,
+                requester: requester.address,
+                beneficiary: beneficiary.address,
+                tag: standardDealTag,
+                prices: ordersPrices,
+                volume: volume,
+                callback: callback,
+                trust: trust,
+                category: category,
+                params: params,
+            });
+            expect(await iexecPoco.balanceOf(proxyAddress)).to.be.equal(0);
```

Comment:
```suggestion
            expect(await iexecPoco.balanceOf(proxyAddress)).to.be.equal(0);
```
& others

Reply:
I'm guessing you wanted to replace `.to.be.equal()` with `.to.equal()` here.
I'll change that and merge but let me know if it's something else.

Reply:
Not really, why do you check balance?

Reply:
I thought about removing these checks but I find it useful to check the initial state. Make sure we are starting from a controlled and known environment.

Reply:
Not sure to follow you :ear: ?